style.css

```
@@ -0,0 +1,67 @@
+@font-face {
+  font-family: 'SBAggroB';
+  src: url('https://cdn.jsdelivr.net/gh/projectnoonnu/noonfonts_2108@1.1/SBAggroB.woff')
+    format('woff');
+  font-weight: normal;
+  font-style: normal;
+}
+@font-face {
+  font-family: 'Pretendard-Regular';
+  src: url('https://cdn.jsdelivr.net/gh/Project-Noonnu/noonfonts_2107@1.1/Pretendard-Regular.woff')
+    format('woff');
+  font-weight: 400;
+  font-style: normal;
+}
+
+body {
+  width: 100vw;
+  height: 100vh;
+  font-family: 'Pretendard-Regular';
+  background-color: rgb(245, 180, 96);
+  display: flex;
+  align-items: center;
+  justify-content: center;
+}
+.container h1 {
+  font-family: 'SBAggroB';
+}
+.container {
+  border-radius: 10px;
+  background-color: white;
+  width: 40rem;
+  height: 50rem;
+  text-align: center;
+  display: flex;
+  flex-direction: column;
+  vertical-align: middle;
```

```suggestion
  justify-content: center;
```
flex-direction 속성을 지정해주었으니 flexbox를 적극 활용해보는 것도 좋겠습니다!